netbox/extras/models/models.py

```diff
@@ -550,7 +551,8 @@ def enqueue_job(cls, func, name, obj_type, user, *args, **kwargs):
             job_id=uuid.uuid4()
         )
 
-        func.delay(*args, job_id=str(job_result.job_id), job_result=job_result, **kwargs)
+        queue = django_rq.get_queue("default")
+        queue.enqueue(func, job_id=str(job_result.job_id), job_result=job_result, **kwargs)
```

Comment:
I don't know enough about django_rq to know if there is a reason to use the job decorator over just enqueuing the job.

I do like that doing this also gives us the ability to schedule jobs down the line.

Reply:
Should have the same net effect either way.